pkg/vulnerability/vulnerability.go

```diff
@@ -130,6 +130,7 @@ func (c Client) getVendorSeverity(vulnID string, vuln *dbTypes.Vulnerability, so
 		return dbTypes.SeverityUnknown.String(), ""
 	}
 
+	log.Warn("Vendor and NVD don't have severity level. Severity from another vendor is used (see `VendorSeverity` in `json` format).", log.String("CVE", vulnID))
```

Comment:
It's not necessarily CVE-ID.

```suggestion
	log.Warn("Vendor and NVD don't have severity level. Severity from another vendor is used (see `VendorSeverity` in `json` format).", log.String("vulnerability-id", vulnID))
```

Reply:
`Vendor` is unclear. We may want to show the source (dbTypes.SourceID).

Reply:
I thought about this, but there is 1 problem with this.
We fill `severity` filed in `trivy-db`:
https://github.com/aquasecurity/trivy-db/blob/b8fe1376ffcdc69fe454f0a8a481ab485e47aea5/pkg/vulnsrc/vulnerability/vulnerability.go#L92-L108

Therefore, we don't have info about vendor for this severity.

We can only check `vendorSeverity` and detect vendor with same severity.
But if 2 vendors use this severity, we may make mistake in our choice

Reply:
I mean, we should say "Debian and NVD don't have severity" when scanning a Debian image and "Ubuntu and NVD don't have severity" when scanning an Ubuntu image. We need to consider what should be displayed with language-specific packages, though.